kuma/wiki/tests/test_admin.py

```diff
@@ -212,15 +216,15 @@ def test_spam_submission_filled(self):
         )
         self.client.login(username='admin', password='testpass')
         response = self.client.get(url)
-        self.assertEqual(response.status_code, 200)
+        assert response.status_code == 200
```

Comment:
Is there a reason to move away from `self.assertEqual` here and in the following unit tests?

I tend to prefer it because it gives more descriptive error messages (showing the bad value) when it fails:

```
Traceback (most recent call last):
  File "test.py", line 13, in test_assertEquals
    self.assertEquals(response.status_code, 200)
AssertionError: 404 != 200
```

versus

```
Traceback (most recent call last):
  File "test.py", line 10, in test_assert
    assert response.status_code == 200
AssertionError
```

```

Reply:
Oh, hey, pytest rewrites bare asserts to get the same kind of nice output messages. Sold.